src/technologies/h.json

```
@@ -1803,9 +1830,10 @@
       "Alpine.js"
     ],
     "pricing": [
-      "onetime"
+      "onetime",
+      "low"
     ],
-    "website": "https://hyva.io"
+    "website": "https://www.hyva.io/hyva-theme-license.html"
```

This isn't usually the licence link but more the product link:

```suggestion
    "website": "https://www.hyva.io/
```

---

The suggested url is the product page for the Theme, and for Checkout I added the checkout-specific page, so I think that qualifies as "product link". Please confirm or reject :)

---

Ah OK. Didn't look closely enough! Resolving.